benchmarks/docker-compose.yml

```diff
@@ -0,0 +1,175 @@
+version: '3'
+services:
+    mlflowdb:
+      image: postgres:13
+      container_name: mlflowdb
+      environment:
+        - POSTGRES_USER=postgres
+        - POSTGRES_PASSWORD=postgres
+        - POSTGRES_DB=mlflow_db
+      networks:
+        - mlflow_network
+      healthcheck:
+        test: ["CMD", "pg_isready", "-U", "postgres"]
+        interval: 5s
+        timeout: 5s
+
+    fasttrackdb:
+      image: postgres:13
+      container_name: fasttrackdb
+      environment:
+        - POSTGRES_USER=postgres
+        - POSTGRES_PASSWORD=postgres
+        - POSTGRES_DB=fasttrack_db
+      networks:
+        - fasttrack_network
+      healthcheck:
+        test: ["CMD", "pg_isready", "-U", "postgres"]
+        interval: 5s
+        timeout: 5s
+
+    mlflowpostgres:
+      image: seeloz/mlflow-server
```

Comment:
is there an official mlflow image we can use?

Reply:
No there isn't unfortunately

Reply:
But if they do add one we can change it